7.1 Snapshot.md

```diff
@@ -0,0 +1,52 @@
++ UTXO: Transactions whose outputs are unspent, remain after the snapshot.
++ Branches: We require that the at the point of pruning only master branch and rejected branch exist.
++ Markers: We require that the part of the Tangle being pruned is not actively dependent on Markers any more. 
++ Approval weight: We require that the approval weight is concluded at a time much before messages reach the pruning age.
++ Voting: We require that any voting is concluded much before messages reach the pruning age. 
++ Data flow: The past cone check must be relaxed (???TODO)
+
+
+## 7.1.3 Proof of Inclusion and Pruning
+
+In order to keep the ledger concise data that is sufficiently far in the past can be removed. We call this process pruning. Thus, the IOTA protocol for a standard node does not attempt to keep all information back until the genesis. Rather the protocol effectively becomes a filter mechanism to proof whether a certain message was at some point in the past included in the ledger. This can be done e.g. through a Merkel tree proof. 
+
+Transactions are a special type of message, that cannot be pruned. Specifically when a transaction contains an unspent output, see [5.1. UTXO](./5.1%20UTXO.md), the transaction shall be kept when pruning the ledger. An optimization of this process is possible by reducing the message to only keeping the information about the unspent outputs, however the implementation will be more complicated, and thus is omitted in this first iteration of the protocol.
+
+## 7.1.5 Pruning dependencies
+
+We require several conditions to be fulfilled in order to allow for a smooth pruning process. Pruning is done when messages reach the age `DELTA_PRUNE`. 
+
+Since messages cannot attach to a message whose timestamp has an age difference of more than `DELTA`, see [4.3.4 R-URTS](./4.3%20Tip%20Selection%20Algorithm.md/#4.3.4%20R-URTS), and the timestamp precision is enforced through voting, see [4.2.3 General Timestamp rules](./4.2%20Timestamps.md/#4.2.3%20General%20Timestamp%20rules). Finalization of messages must be guaranteed to happen before the point of pruning. Thus, any message must be either confirmed or rejected before the elapse of the pruning age. This requires that both voting on timestamps and conflicts, as well as the accumulation of sufficient approval weight takes a time much smaller than `DELTA_PRUNE`. 
+
+In order to allow a smooth pruning, any message at the age `DELTA_PRUNE` must either belong to the branch `masterBranch` or `rejectedBranch`. This simply means that messages are either confirmed or rejected. 
+
+Transaction must pass a past cone check, see [### 2.4.4.6 Booker - Point 7.](./2.4%20Data%20Flow.md/#2.4.4.6%20Booker). However the past cone check cannot apply to messages below a certain age `DELTA_PASTCHECK<<DELTA_PRUNE`. Thus, the past cone check there needs to be relaxed (???TODO).
+
+With the above conditions we can safely assume any message before the time `DELTA_PRUNE>>DELTA` can be pruned. 
```

Comment:
From the text, its not clear for me the requirements for a message to be pruned and why the checks described satisfy it. Perhaps we could organize the requirements in itens? It should solve most of this.

Reply:
```suggestion
With the above conditions we can safely assume any message before the time `DELTA_PRUNE>>DELTA` can be pruned. In summary this is for the following reasons:
- Timestamps are sufficiently accurate.
- Messages can only create parent-child relationships up to a predefined age.
- Branches are cleaned up at the depth of pruning, i.e. all messages that have pruning age are either confirmed or rejected.
- Transactions are only removed once all of their outputs are confirmed spent. 
```

Reply:
This is a really important point. Could you check if the above made this more clear? @GalRogozinski @darcy-camargo

Reply:
Gave a bit more information on the age restriction. Also there is any reason you are avoiding using the word "pruning" with value messages?

```suggestion
With the above conditions we can safely assume any message before the time `DELTA_PRUNE>>DELTA` can be pruned. In summary this is for the following reasons:
- Timestamps are sufficiently accurate.
- Messages can only create parent-child relationships up to a predefined age that is more recent than the messages timestamps.
- Branches are cleaned up at the depth of pruning, i.e. all messages that have pruning age are either confirmed or rejected.
- Transactions are only pruned once all of their outputs are confirmed spent. 

```

Reply:
imo you could make the "requirements" and "explanations" before this line also in topics, would help with reading.

Reply:
@darcy-camargo  i wasn't aware i was avoiding it. Could you point me to a line where this is the case?
 also added another bullet list for clarity.